lib/Components/Cart/cart.css

```diff
@@ -0,0 +1,211 @@
+@import "/lib/Components/core.css";
+@import "/lib/Components/utilities.css";
+@import "/lib/Components/Header/header.css";
+@import "/lib/Components/Footer/footer.css";
+
+:root {
+    --COLOR-RED-1: 255, 0, 0;
```

Comment:
Explanation - I'm not able to comprehend the use of this variable. Can you please elaborate? Some reference blog/docs would be a great help